controller/workitem.go

```diff
@@ -53,6 +55,9 @@ type WorkItemControllerConfig interface {
 	GetCacheControlWorkItem() string
 }
 
+// uuidStringCache stores the UUID cache for ID resolvings of the CSV conversion
+var uuidStringCache = make(map[string]string)
```

Comment:
I highly suggest that you secure access to your cache by using a mutex and that you establish a way to clear the cache as well? What happens if an iteration is renamed for example? Will the cache be the same or how will it reflect the change?

```suggestion
// UUIDStringCache caches IDs of objects as keys and the objects resolved to a string as the values
type UUIDStringCache struct {
	cache      map[string]string
	mapLock sync.RWMutex
}
```

Reply:
The cache is only valid for one request. It caches repeated access to labels, areas, iterations, anything that will otherwise be queried possibly hundreds of times. I would not do a cache outside of the request scope, that would indeed require a lot more sync.

Reply:
Fine but currently the cache is defined on package level and not on request level. Please put it inside a function then.

Reply:
Correct, moved this to the function. Fixed d3bfe57.